enterprise/cli/licenses_test.go

```diff
@@ -0,0 +1,182 @@
+package cli_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/cli/clitest"
+	"github.com/coder/coder/coderd/coderdtest"
+	"github.com/coder/coder/codersdk"
+	"github.com/coder/coder/enterprise/cli"
+	"github.com/coder/coder/enterprise/coderd"
+	"github.com/coder/coder/pty/ptytest"
+	"github.com/coder/coder/testutil"
+)
+
+const fakeLicenseJWT = "test.jwt.sig"
+
+func TestLicensesAddSuccess(t *testing.T) {
+	t.Parallel()
+	// We can't check a real license into the git repo, and can't patch out the keys from here,
+	// so instead we have to fake the HTTP interaction.
```

Comment:
So that we can test licenses for real (in CI) should we consider adding a secret in GitHub actions and only test it in CI (or when defined locally?).

I'm not sure how it would work, just throwing it out there.

Reply:
Yeah, if we want to build e2e tests, we'll need real keys that we don't check into the repo